test/test_conflict_ui.py

```
@@ -0,0 +1,504 @@
+    def test_interactive_conflict_resolution_skip(self, mock_stdout, mock_input):
+        """Test skipping package during conflict resolution."""
+        # Simulate user choosing to skip (option 3)
+        mock_input.side_effect = ['3']
+        
+        conflicts = [
+            ('nginx', 'apache2')
+        ]
+        
+        # Should exit on choice 3
+        with self.assertRaises(SystemExit):
+            result = self.cli._resolve_conflicts_interactive(conflicts)
```

Ruff (0.14.6)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)